x/marketmaker/keeper/proposal_handler.go

```diff
@@ -0,0 +1,197 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/cosmosquad-labs/squad/v2/x/marketmaker/types"
+)
+
+// HandleMarketMakerProposal is a handler for executing a market maker proposal.
+func HandleMarketMakerProposal(ctx sdk.Context, k Keeper, proposal *types.MarketMakerProposal) error {
+	if proposal.Inclusions != nil {
+		if err := k.InclusionMarketMakers(ctx, proposal.Inclusions); err != nil {
+			return err
+		}
+	}
+
+	if proposal.Distributions != nil {
+		if err := k.DistributionMarketMakerIncentives(ctx, proposal.Distributions); err != nil {
+			return err
+		}
+	}
+
+	if proposal.Exclusions != nil {
+		if err := k.ExclusionMarketMakers(ctx, proposal.Exclusions); err != nil {
+			return err
+		}
+	}
+
+	if proposal.Rejections != nil {
+		if err := k.RejectionMarketMakers(ctx, proposal.Rejections); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// InclusionMarketMakers is a handler for include applied and not eligible market makers.
+func (k Keeper) InclusionMarketMakers(ctx sdk.Context, proposals []types.MarketMakerHandle) error {
+	for _, p := range proposals {
+		mmAddr, err := sdk.AccAddressFromBech32(p.Address)
+		if err != nil {
+			return err
+		}
+		mm, found := k.GetMarketMaker(ctx, mmAddr, p.PairId)
+		if !found {
+			return sdkerrors.Wrapf(types.ErrNotExistMarketMaker, "%s is not a applied market maker", p.Address)
+		}
+		// pass already eligible market maker
+		if mm.Eligible {
+			return sdkerrors.Wrapf(types.ErrInvalidInclusion, "%s is already eligible market maker", p.Address)
+		}
```

Comment:
Should we remove the comment `// pass already eligible market maker`? I think the comment refers to use `continue` here to pass already eligible market maker, but it actually returns an error.